### PR TITLE
Remove duplicate tests

### LINKED ISSUE #234: Check tests for duplicates.
It is possible that some tests are repeated several times without any need. If so, then we need to fix it.

### PR DESCRIPTION
**Pull request by artiemq**
Fix #234 

Should i also refactor some tests to make them consistent with others?

E.g create variable for the result of the tested function [here](https://github.com/lk-geimfari/mimesis/blob/master/tests/test_data/test_cryptographic.py#L32L33)

### CODE REVIEW

**Comment by lk-geimfari** [comment]
@artiemq Sure!

**Comment by lk-geimfari** [comment]
@artiemq Do you will work on refactoring in this branch or i can merge it?

**Comment by artiemq** [comment]
I think it would be better to work in this branch

**Comment by lk-geimfari** [comment]
@artiemq Okay! Let me know when you finish.

**Comment by lk-geimfari** [comment]
@artiemq The tests failed? Any ideas why?

**Comment by artiemq** [comment]
@lk-geimfari looks like the release of `flake8==3.5.0` broke `pytest-flake8` tholo/pytest-flake8#34

**Comment by lk-geimfari** [comment]
@artiemq Oh, god. I think we should to fix versions of dependencies.

**Comment by artiemq** [comment]
@lk-geimfari Ready for review

File: .travis.yml [hunk 1]
```diff
@@ -9,13 +9,7 @@ python:
...
   - "3.7-dev"
 
-install:
-  - pip install codecov
-  - pip install pytest-cov
-  - pip install flake8-builtins
-  - pip install flake8-commas
-  - pip install flake8-quotes
-  - pip install pytest-flake8
+install: pip install -r requirements.txt
```
Comment (lk-geimfari) [inline .travis.yml]: Rename it to `dev_requirements.txt`, please. In order to avoid misunderstandings.

**Pull request merged**